packages/alfa-rules/test/sia-er62/rule.spec.tsx

```diff
@@ -1070,229 +1225,255 @@ test(`evaluate() passes an <a> element that is distinguishable from the <p> pare
         ["outline", "0px"],
         [
           "background",
-          "linear-gradient(to right, rgb(100% 0% 0%) 20%, rgb(100% 64.70588000000001% 0%) 40%, rgb(100% 100% 0%) 60%, rgb(0% 50.196079999999995% 0%) 100%)",
+          "linear-gradient(to right, rgb(0% 0% 81.96078%) 50%, rgb(0% 0% 25.88235%) 50%) 0% 0%",
         ],
+        ["color", "rgb(100% 100% 100% / 10%)"],
       ],
       contrastPairings
     )
   );
 
   t.deepEqual(await evaluate(ER62, { document }), [
-    passed(ER62, target, {
-      1: Outcomes.IsDistinguishable([style], [style], [style]),
+    failed(ER62, target, {
+      1: Outcomes.IsNotDistinguishable([noStyle], [noStyle], [noStyle]),
     }),
   ]);
 });
 
-test(`evaluate() passes an <a> element that is distinguishable from the <p> parent element as some foregrounds have a contrast of 3:1 in the parent element`, async (t) => {
+/******************************************************************
+ *
+ * Inapplicable tests
```

Comment:
We should also have an applicable test where the link has the same colour as **part of** the container, but not all of it.